### PR TITLE
Include registration statuses to event items

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

Makes it much easier to differentiate between the list of dates in the items.
There's no need to show the activation time for registrations if the user has
already registered.

Besides, people prefer to use the event list on /events rather than the
one on the user profile to view upcoming events.

# Result

- [x] Changes look good on both light and dark theme.
- [x] Changes look good with different viewports (mobile, tablet, etc.).
- [x] Changes look good with slower Internet connections.

Look at _Deloitte AS_ and _Fadderpåmelding Kommunikasjonsteknologi_ to see event items with 2 dates ... difficult to differentiate between the two:

<table>
    <tr>
        <td>Before</td>
        <td>After</td>
    </tr>
    <tr>
        <td>
            image
        </td>
        <td>
            image
        </td>
    </tr>
</table>

# Testing

- [x] I have thoroughly tested my changes.

Tested all event types, and the three different registration statuses (unregistered, waiting list and registered/admitted)

### CODE REVIEW

File: app/components/EventItem/index.tsx [hunk 1]
```diff
@@ -205,12 +201,20 @@ const EventItem = ({
...
         >
-          <Flex column gap="var(--spacing-xs)" className="secondaryFontColor">
-            <div>
-              <h3 className={styles.eventItemTitle}>{event.title}</h3>
-              {event.totalCapacity > 0 && <Attendance event={event} />}
-            </div>
-            <TimeStartAndRegistration event={event} />
+          <Flex column gap="var(--spacing-sm)" className="secondaryFontColor">
+            <h3 className={styles.eventItemTitle}>{event.title}</h3>
+            {event.totalCapacity != null && event.totalCapacity > 0 && (
```
Comment (norbye) [inline app/components/EventItem/index.tsx:206]:
```suggestion
            {event.totalCapacity !== null && event.totalCapacity > 0 && (
```
I like ===

**Review by norbye — APPROVED** [review]
Love it!! Have been missing a nice overview of events I've registered to for so long

File: app/components/EventItem/index.tsx (hunk 1 above)
Comment (ivarnakken) [reply app/components/EventItem/index.tsx:206]: This is actually intentional so to include `undefined` 😄

Comment (norbye) [reply app/components/EventItem/index.tsx:206]: Ah, I thought `!=` would cover more than just `undefined` and `null`, but it seems I was mistaken

**Pull request merged**